src/backend/renderer/element/mod.rs

```diff
@@ -350,7 +348,7 @@ pub trait RenderElement<R: Renderer>: Element {
     ) -> Result<(), R::Error>;
 
     /// Get the underlying storage of this element, may be used to optimize rendering (eg. drm planes)
-    fn underlying_storage(&self, renderer: &R) -> Option<UnderlyingStorage<'_, R>> {
+    fn underlying_storage(&self, renderer: &mut R) -> Option<UnderlyingStorage> {
```

Comment:
AFAICT, it was uneccessary to make this `&mut R`. I've not tried all build options, but with default build I can get rid of it locally.

Reply:
Yes, `&mut R` was necessary for exporting a texture to a dmabuf which got removed from this PR. But I think it would be fine to leave it `&mut R` to allow such a scenario without breaking the API again
@Drakulix any objections against making it `&mut`  as part of this PR?

Reply:
Nope. Feel free to keep it.